src/OgGroupAudienceHelper.php

```diff
@@ -24,6 +24,15 @@ class OgGroupAudienceHelper {
    */
   const DEFAULT_FIELD = 'og_group_ref';
 
+  /**
+   * The name of the field which reference user to groups.
+   */
+  const USER_REFERENCE_FIELD = 'og_membership_reference';
```

Comment:
This is not directly related to this PR, but why do we still use this field? We now have the OgMembership entity which tracks which groups a user is a member of. We don't need this field any more.